scripts/create-stonks-instances.ts

```diff
@@ -0,0 +1,116 @@
+import { assert } from 'chai'
+import { network } from 'hardhat'
+
+import fmt from '../utils/format'
+import { confirmOrAbort } from '../utils/prompt'
+import { getDeployer, verify } from '../utils/deployment'
+import { StonksFactory__factory } from '../typechain-types'
+import { StonksDeployedEvent } from '../typechain-types/contracts/factories/StonksFactory'
+
+interface StonksConfig {
+  tokenFrom: string
+  tokenTo: string
+  orderDurationInSeconds: bigint
+  marginBasisPoints: bigint
+  priceToleranceInBasisPoints: bigint
+}
+
+const AGENT = ''
+const STONKS_FACTORY = ''
+const AMOUNT_CONVERTER = ''
+const MANAGER_ADDRESS = ''
+const STONKS_CONFIGS: Record<string, StonksConfig> = {}
+
+async function main() {
+  // prettier-ignore
+  console.log(
+    `Preparing for ${fmt.name('Stonks')} contracts deployment on "${fmt.network(network.name)}" network,`,
+    `using the ${fmt.name("StonksFactory")} on address ${fmt.address(STONKS_FACTORY)} ...\n`
+  )
+
+  const deployer = await getDeployer()
+
+  for (const [pair, config] of Object.entries(STONKS_CONFIGS)) {
+    console.log(`Deploy config for pair ${fmt.name(pair)}:`)
+    console.log(`  * token from: ${fmt.value(config.tokenFrom)}`)
+    console.log(`  * token to: ${fmt.value(config.tokenTo)}`)
+    console.log(`  * order duration (sec): ${fmt.value(config.orderDurationInSeconds)}`)
+    console.log(`  * margin (bps): ${fmt.value(config.marginBasisPoints)}`)
+    console.log(`  * price tolerance (bps): ${fmt.value(config.priceToleranceInBasisPoints)}`)
```

Comment:
Those names are pretty hard to memorize (i.e., which one is the upper bound or lower), I would suggest disambiguating them a bit with additional printable hints